src/core-app-plane/bash-job-runner.ts

```diff
@@ -105,17 +163,15 @@ export class BashJobRunner extends Construct {
       }
     }
 
-    this.environmentVariablesToOutgoingEvent = props.environmentVariablesToOutgoingEvent;
-
     const codeBuildProjectEncryptionKey = new kms.Key(
-      scope,
+      this,
       `${props.name}-codeBuildProjectEncryptionKey`,
       {
         enableKeyRotation: true,
       }
     );
 
-    this.codebuildProject = new codebuild.Project(scope, `${props.name}-codebuildProject`, {
+    const codebuildProject = new codebuild.Project(this, `${props.name}-codebuildProject`, {
```

Comment:
i think we can get rid of the `name` argument for bash-job-runners. I originally had it to make the construct ids unique, but if they are being built as part of their own constructs, that shouldn't be a problem.